client/state/initial-state.js

```diff
@@ -81,7 +100,7 @@ function addSympathy( initialStateLoader ) {
 	return () => createReduxStore( getInitialServerState() );
 }
 
-const loadInitialState = addSympathy( initialState => {
+const loadInitialState = maybeAddSympathy( initialState => {
```

Comment:
Nitpick: I slightly prefer the old name that's not prefixed with `maybe`.

Reply:
I'd like to clearly communicate that this function *doesn't* always `addSympathy`.

Reply:
@nylen Can we avoid the `maybe` naming? I would be fine with something more verbose.